### PR TITLE
Add telemetry

### LINKED ISSUE #63: Add telemetry

### PR DESCRIPTION
**Pull request by jeffmaury**
Fixes #63

Signed-off-by: Jeff MAURY <jmaury@redhat.com>

### CODE REVIEW

**Comment by jeffmaury** [comment]
@odockal @jrichter1 any clue with UI tests are failing ?

**Comment by odockal** [comment]
@jeffmaury Builds on Travis were disabled due to "negativa credit balance" on public repos. So I think that removing travis is quite necessary.

File: package.json [hunk 1]
```diff
@@ -125,6 +125,7 @@
         "@types/yauzl": "^2.9.0",
         "request": "^2.47.0",
         "request-promise-native": "^1.0.7",
-        "yauzl": "^2.10.0"
+        "yauzl": "^2.10.0",
+        "@redhat-developer/vscode-redhat-telemetry": "0.0.11"
```
Comment (dgolovin) [inline package.json:129]: Update telemetry module to 0.0.13.

File: src/extension.ts [hunk 2]
```diff
@@ -74,45 +78,59 @@ export function filterCatalogForRuntimes(catalog: any, runtimeIds:string[]) {
...
+                                    }
                                 }
                             }
                         }
                     }
                 }
             }
         }
+    } catch (error) {
+        telemetryProps.error = error.toString();
```
Comment (dgolovin) [inline src/extension.ts:130]: make sure that user sensitive information cannot leak through exception message. For example `catalogBuilder.zip` method could fail and have local path reported as part of the exception's message.

**Review by dgolovin — CHANGES_REQUESTED** [review]

**Comment by jeffmaury** [comment]
Replaced by #68

**Pull request merged**